subprojects/solver/solver-z3/src/main/java/hu/bme/mit/theta/solver/z3/Z3TermTransformer.java

```diff
@@ -264,7 +264,7 @@ private Expr<?> transformFpLit(final com.microsoft.z3.Expr term) {
         BigFloat bigFloat = new BigFloat((fpTerm).getSignificand(),
                 FpUtils.getMathContext(type, FpRoundingMode.RNE)).multiply(
                 new BigFloat("2", FpUtils.getMathContext(type, FpRoundingMode.RNE)).pow(
-                        new BigFloat((fpTerm).getExponent(),
+                        new BigFloat((fpTerm).getExponent(true),
```

Comment:
Is this a bugfix?

Reply:
This changed from 4.5.0 to 4.13.0, it does not have a 0-argument version anymore. As far as I could tell, this is the version of the call that does not change its semantics -- but I couldn't find much info on this topic, so a quick cross-check would be appreciated. (This is the `biased` param of the call).

Reply:
Yes, indeed.